akka-persistence-query/src/main/scala/akka/persistence/query/scaladsl/CurrentDurableStatePersistenceIdsQuery.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2015-2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.query.scaladsl
+
+import akka.NotUsed
+import akka.persistence.state.scaladsl.DurableStateStore
+import akka.stream.scaladsl.Source
+
+/**
+ * A plugin may optionally support this query by implementing this trait.
+ */
+trait CurrentDurableStatePersistenceIdsQuery[A] extends DurableStateStore[A] {
+
+  /**
+   * Get the current persistence ids.
+   *
+   * Not all durable state plugins may support in database paging, and may simply use drop/take Akka streams operators
+   * to manipulate the result set according to the paging parameters.
+   *
+   * @param afterId The ID to start returning results from, or [[None]] to return all ids. This should be an id
+   *                returned from a previous invocation of this command. Callers should not assume that ids are
+   *                returned in sorted order.
+   * @param limit The maximum results to return. Use Long.MaxValue to return all results. Must be greater than zero.
+   * @return A source containing all the persistence ids, limited as specified.
+   */
+  def currentPersistenceIds(afterId: Option[String], limit: Long): Source[String, NotUsed]
```

Comment:
@jroper was there a reason for why we are not adding the same for event sourced?
maybe @raboof already asked this and I just missed the answer?

Reply:
No reason other than I thought they were different enough to warrant being done in separate PRs.